maintenance_test.py

```diff
@@ -36,33 +36,36 @@ def test_drain(self):
         Drain a node an restart it.
         """
         self.db_cluster.add_nemesis(DrainerMonkey)
-        # this nemesis is not periodic and will do
-        # the stop and restart
-        time.sleep(10 * 60)
```

Comment:
After removing the time.sleep(), time module is no longer used. 
Travis-CI test failed for "W0611: 16,0: : Unused import time"
https://travis-ci.org/scylladb/scylla-cluster-tests/builds/144050246